src/components/BuildLogPage/BuildLogPage.tsx

```diff
@@ -29,7 +42,31 @@ export const BuildLogPage = () => {
     }
   }, [serviceContainerBuildLogRunner, buildId, isBuilding]);
 
-  const logActions = [<BuildLogLink key="log-link" buildId={buildId!} />];
+  useEffect(() => {
+    if (
+      (buildBelongToCurrentUser || isCurrentUserAdmin) &&
+      serviceContainerBuild.data?.status &&
+      buildStatusData[serviceContainerBuild.data.status].failed
+    ) {
+      serviceContainerBuildSshCredentialsRunner({ serviceData: { id: buildId } });
+    }
+  }, [
+    serviceContainerBuildSshCredentialsRunner,
+    buildId,
+    buildBelongToCurrentUser,
+    serviceContainerBuild.data?.status,
+    isCurrentUserAdmin,
+  ]);
+
+  const logActions = [
+    <SshCredentialsButton
```

Comment:
The SSH credentials button should no longer be on the Build log page, but rather on top bar of Build pages